direct/engine.py

```diff
@@ -167,6 +169,13 @@ def predict(
         self.ndim = dataset.ndim  # type: ignore
         self.logger.info("Data dimensionality: %s.", self.ndim)
 
+        if self.ndim == 3 and batch_size > 1:
+            warnings.warn(
+                f"Batch size for inference of 3D data must be 1. Received {batch_size}."
+                f"`batch_size` overwritten by 1."
+            )
+            batch_size = 1
+
```

Comment:
Why must it be one? Perhaps this could become more efficient if we can increase it.

Reply:
Removed